CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ onCameraChangeListener(CameraChangedEventData data) {
   print("CameraChangedEventData: timestamp: ${data.timestamp}, cameraState: ${data.cameraState}");
 }
 ```
+* Print to console native Maps SDK logs in debug configuration.
+Logs are proxied only in debug configuration and can be disabled completely by passing environment flag `MAPBOX_LOG_DEBUG` with false value.
+
+# 2.3.0
```

Comment:
should this header be above to include entries from log debug and camera state observe?